### PR TITLE
Fix/add pages

### PR DESCRIPTION
**Pull request by ankitmlesmagico**

### CODE REVIEW

File: docs/pages/otpPage.md [hunk 1]
```diff
@@ -79,45 +82,50 @@ The component's theme colors can be easily customized by updating the values in
...
 
 ```json
 "component": {
-  "allowOverride": false,
-  "title": "Welcome to OTP Verification!"
+     "allowOverride": false,
+     "logo": "https://seeklogo.com/images/C/corporate-company-logo-749CEE6ADC-seeklogo.com.png",
+     "showLogo": false,
+     "otpLength": 4,
+     "resendOtpTimer": "30",
```
Comment (biratdatta) [inline docs/pages/otpPage.md]: "resendOtpTimer": 30,

File: docs/molecules/chatui.md [hunk 2]
```diff
@@ -50,28 +50,18 @@ The ChatUI component offers flexibility for customization through its configurat
...
-      "transliterationApi": "",
-      "transliterationInputLanguage": "",
-      "transliterationOutputLanguage": "",
-      "transliterationProvider": "",
-      "transliterationSuggestions": 3,
-      "rightAction": null,
-      "leftAction": null
-  
+      "transliterationProvider": "bhashini",
+      "transliterationSuggestions": "3",
```
Comment (biratdatta) [inline docs/molecules/chatui.md]: "transliterationSuggestions": 3,

**Review by biratdatta — CHANGES_REQUESTED** [review]
Quality of Life changes because it's better to store numeric values as numbers rather than strings.

File: docs/pages/otpPage.md (hunk 1 above)
Comment (ankitmlesmagico) [reply docs/pages/otpPage.md]: done

File: docs/molecules/chatui.md (hunk 2 above)
Comment (ankitmlesmagico) [reply docs/molecules/chatui.md]: done

File: docs/stencilweb/overview.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+title: Stencil Web
+sidebar_label: Overview
+---
+
+<head>
+  <title> Overview </title>
+  <meta name="description" content="your meta content goes here" />
+</head>
+
+A `Stencil Web` is a libraray made up of various molecules and pages. It represents a more intricate UI element designed to fulfill a specific function within an application's interface. For example, a JSON to Table or an OTP (One-Time Password) Input Molecule, each composed of multiple atoms to provide a comprehensive and functional unit. Similarly, Otp Page, Login Page, and other pages provide complete interfaces for common application needs.
```
Comment (prtkjakhar) [inline docs/stencilweb/overview.md]: `library` spell error

**Comment by ankitmlesmagico** [comment]
done

File: docs/stencilweb/overview.md (hunk 3 above)
Comment (ankitmlesmagico) [reply docs/stencilweb/overview.md]: done

**Pull request merged**